dataload/import_to_elasticsearch.py

```diff
@@ -14,12 +14,18 @@
 
 import sys
 
+from django.core.cache import cache
+import django
+
+
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "grantnav.settings")
 
+
 sys.path.append(os.path.dirname(os.path.dirname(os.path.realpath(__file__))))
 
 from grantnav.frontend.org_utils import new_ordered_names, new_org_ids, get_org, OrgNotFoundError # noqai
 
+django.setup()
```

Comment:
Can `django.setup()` safely go above the `from grantnav.frontend ... import ...` line, or does it need some env setup from that import?

I say because I once spent ages debugging the order of `django.setup()` and imports, where some imports wouldn't work properly until after setup, and since then as a general rule I've tried to put it as high up the file as possible

Reply:
Yep happy to do that